index.js

```diff
@@ -6,21 +6,24 @@
 
 var Path = require('path');
 var Hapi = require('hapi');
+var vision = require('vision');
+var inert = require('inert');
 
 var handlebars = require('handlebars');
 handlebars = require('handlebars-helper-sri').register(handlebars);
 
 var helpers = require('./lib/helpers.js');
 
-var CSP_HEADER = "default-src 'none'; img-src 'self'; style-src 'self'; font-src 'self' ; frame-src 'self'"; // jshint ignore:line
-
 var server = new Hapi.Server();
+
+var CSP_HEADER = 'default-src "none"; img-src "self"; style-src "self"; font-src "self"; child-src "self"';
```

Comment:
We care about webcompat and users. That's kind of our thing :stuck_out_tongue: 

Which browsers are that and are they among the auto-update browsers?
I'd suppose Safari/Webkit on OS X and iOS devices.